pilot.go

```diff
@@ -142,7 +141,10 @@ var _ autopilot.ChannelController = (*chanController)(nil)
 // and all interfaces needed to drive it won't be launched before the Manager's
 // StartAgent method is called.
 func initAutoPilot(svr *server, cfg *autoPilotConfig) (*autopilot.ManagerCfg, error) {
-	atplLog.Infof("Instantiating autopilot with cfg: %v", spew.Sdump(cfg))
+	atplLog.Infof("Instantiating autopilot with max_channels=%d, "+
```

Comment:
Nit: there's a trailing comma here while the other params don't have one. Also I think we have a preference for using camel case for stuff like this, though it's not strictly enforced.

Reply:
What in the config spews a curve?

Reply:
commas added.

> What in the config spews a curve?

There isn't one, this is just to get rid of any spews in info logs since it breaks parsing in the app, and generally make everything more consistent